covid19-dashboard/README.md

```diff
@@ -1,15 +1,61 @@
-# dc-covid19-website
-Dashboard to display the top number of COVID-19 regions (State or County) in the USA by cases or deaths.
+# DC COVID-19 Dashboard
+NOTE: ```place_type``` refers to either ```Country, State or County```.
 
-To build the project:
-1. Create a config.py file in the covid19-dashboard directory.
-   a. The config file should contain your DataCommons API Key as follows:``API_KEY="YOURAPIKEY"``.
-2. Go to dc-covid19 and execute ``npm run build`` to build the static React files.
+Examples:
+get_stats(place_type="Country") -> Returns stats for all Countries.
+get_stats(place_type="State") -> Returns stats for all US States.
+get_stats(place_type="County") -> Returns stats for all US Counties.
 
-To deploy the project to Google Cloud App Engine:
-1. Install Google Cloud Command Line Tools.
-2. Set Up an App Engine Instance.
-3. Login to your App Engine Instance using the Google Cloud Command Line Tools.
-4. Execute ``gcloud app deploy``.
-5. Execute ``gcloud app deploy cron.yaml``.
 
+#### Server APIs:
+The APIs are based on a ```place_type```.
+
+**api/country-data**: returns the data for all Countries in the world with WHO COVID-19 data.
+ - downloads in a few ms.
```

Comment:
These statements are hard to stand on it's own - with what connection speed, what was the server processing time? etc.

What stands is the size of the returned payload, and perhaps server wall time. Just payload size should suffice though.

Reply:
Fixed.